readme.md

```diff
@@ -32,12 +32,18 @@ Source object to copy properties from.
 
 #### mapper
 
-Type: `Function`
+Type: `(sourceKey, sourceValue, source) => [targetKey, targetValue, mapperOptions?]`
 
-Mapping function.
+##### mapperOptions
 
-- It has signature `mapper(sourceKey, sourceValue, source)`.
-- It must return a two item array: `[targetKey, targetValue]`.
+Type: `object`<br>
+Default: `{shouldRecurse: true}`
+
+######  shouldRecurse
+
+Type: `boolean`<br>
+
+Stop recursing down the tree if `false`.
```

Comment:
I think this should be written in a more general form. It shouldn't say what it does when false, but rather what it does at all. "Down the tree" is also not clear enough, as each call only decides for the upcoming level.